src/util/strencodings.cpp

```diff
@@ -81,16 +81,19 @@ template <typename Byte>
 std::optional<std::vector<Byte>> TryParseHex(std::string_view str)
 {
     std::vector<Byte> vch;
-    auto it = str.begin();
-    while (it != str.end()) {
+    vch.reserve(str.size() / 2);
```

Comment:
I think for the cleanest diff the second commit could only have just this one line, no?

Reply:
Done, thanks